### ISSUE TITLE
2018 data file?

### ISSUE THREAD

**Issue by cstewartiii** [openelections/openelections-results-wv#1]
Was there a 2018 data file that has been pulled?  At MEDSL we are working on a file named 20181106__wv__general__precinct.csv, which is the openelections naming convention, but I don't see the file in the current github.  I'm searching for the 2018 file because I've found an issue with (at least) some of the returns pertaining to the 2nd CD, and I'm trying to track down the source of the data.

**Comment by dwillis** [comment]
@cstewartiii Yes - because WV changed the way it published election results, we had to convert its data in a different way than we did for earlier elections. The file is at https://github.com/openelections/openelections-data-wv/blob/master/2018/20181106__wv__general__precinct.csv. Please let me know about the issues you're seeing, and we'll fix.

**Comment by cstewartiii** [comment]
Thanks.  Sorry I missed that.  What I've noticed (and have confirmed in the file you pointed me to) is that Talley Sergent (CD2 Dem candidate) is coded as being in the Senate race in  Hampshire, Lewis, and Upshur counties.  There may be some other mis-codings (we're doing some data cleaning right now as we convert this file to our standard format), and I'll let you know if we find any more.

**Comment by dwillis** [comment]
@cstewartiii Those individual county files and the statewide file have been fixed and pushed.
